styles/components/_form.scss

```diff
@@ -55,8 +63,8 @@ textarea,
   line-height: 40px;
 }
 
-.medium-editor-element {
-  padding-top: $padSm;
+.trumbowyg div[contenteditable].rows6 {
```

Comment:
If the rows6 class is only used for the trumbowyg div, it would be better to put this on .trumbowyg-editor instead, and remove the rows6 class entirely, rather than adding a new class.

If the height should only be applied to the editor in the edit listing view, not all instances of the editor, then this rule should be in the editListing.scss file. 

It looks like trumbowyg sets its own height in the style attribute, so it would be best to just remove this rule, since it is overridden by trumbowyg.